initfiles/componentfiles/configxml/@temp/esp_service_wslogging.xsl

```diff
@@ -19,56 +38,25 @@ xmlns:seisint="http://seisint.com"  xmlns:set="http://exslt.org/sets" exclude-re
-                </xsl:if>
-
-                <xsl:if test="not($managerNode/ESPLoggingAgent[1]/@ESPLoggingAgent)">
-                     <xsl:message terminate="yes">ESP Logging Agent <xsl:value-of select="$managerNode/@ESPLoggingAgent[1]"/>is undefined!</xsl:message>
-                </xsl:if>
-
-                <LoggingManager name="{$managerNode/@name}">
-
-                <xsl:for-each select="$managerNode/ESPLoggingAgent">
-                    <xsl:variable name="agentName" select="@ESPLoggingAgent"/>
-                    <xsl:variable name="agentNode" select="/Environment/Software/ESPLoggingAgent[@name=$agentName]"/>
-
-                    <xsl:if test="not($agentNode)">
-                        <xsl:message terminate="yes">An ESP Logging Agent <xsl:value-of select="$agentName"/>  for <xsl:value-of select="$managerNode/@name"/> is undefined!</xsl:message>
-                    </xsl:if>
-
-                    <xsl:if test="string($agentNode/@ESPServer) = ''">
-                        <xsl:message terminate="yes">ESP server is undefined for <xsl:value-of select="$agentName"/> </xsl:message>
-                    </xsl:if>
-
-                    <xsl:variable name="espServer" select="$agentNode/@ESPServer"/>
-                    <xsl:variable name="espNode" select="/Environment/Software/EspProcess[@name=$espServer]"/>
-
-                    <xsl:if test="not($espNode)">
-                        <xsl:message terminate="yes">ESP process is undefined!</xsl:message>
-                    </xsl:if>
-
-                    <xsl:variable name="espPort" select="$espNode/EspBinding[@service='wslogging']/@port"/>
-
-                    <xsl:if test="string($espPort) = ''">
-                        <xsl:message terminate="yes">ESP server port is undefined for <xsl:value-of select="$espServer"/>!</xsl:message>
-                    </xsl:if>
-
-                    <xsl:variable name="espNetAddress" select="$espNode/Instance/@netAddress"/>
-
-                    <xsl:if test="string($espNetAddress) = ''">
-                        <xsl:message terminate="yes">ESP NetAddress is undefined!</xsl:message>
-                    </xsl:if>
-                    <xsl:variable name="wsloggingUrl"><xsl:text>http://</xsl:text><xsl:value-of select="$espNetAddress"/><xsl:text>:</xsl:text><xsl:value-of select="$espPort"/></xsl:variable>
-                        <LogAgent name="{$agentName}" type="LogAgent" services="GetTransactionSeed,UpdateLog" plugin="espserverloggingagent">
-                            <ESPServer url="{$wsloggingUrl}" user="{$agentNode/@User}" password="{$agentNode/@Password}"/>
-                            <xsl:if test="string($agentNode/@MaxServerWaitingSeconds) != ''">
-                                <MaxServerWaitingSeconds><xsl:value-of select="$agentNode/@MaxServerWaitingSeconds"/></MaxServerWaitingSeconds>
-                            </xsl:if>
+            <xsl:if test="string(CassandraLoggingAgent[1]/@name) = ''">
```

Comment:
Just noticed this test is duplicated in LN... we might want to componetize into individual .xls files and include them into esp_service_wslogging.xsl